src/status_im2/contexts/wallet/receive/view.cljs

```diff
@@ -0,0 +1,102 @@
+(ns status-im2.contexts.wallet.receive.view
+  (:require
+    [quo.core :as quo]
+    [react-native.blur :as blur]
+    [react-native.core :as rn]
+    [react-native.platform :as platform]
+    [react-native.safe-area :as safe-area]
+    [react-native.share :as share]
+    [reagent.core :as reagent]
+    [status-im2.common.qr-codes.view :as qr-codes]
+    [status-im2.constants :as constants]
+    [status-im2.contexts.wallet.common.sheets.network-preferences.view :as network-preferences]
+    [status-im2.contexts.wallet.receive.style :as style]
+    [utils.i18n :as i18n]
+    [utils.image-server :as image-server]
+    [utils.re-frame :as rf]))
+
+(def id-to-network
+  {constants/mainnet-chain-id  :ethereum
```

Comment:
can we please avoiding to re add this data. We already have it in the sub `:walet/network-details` and it's nicer to keep in one place.

Reply:
or add a new sub with these details based from network-details etc 👍

Reply:
@J-Son89 `id-to-network` is a map. The sub `:walet/network-details` returns a sequence. I think it is cleaner to match using a map than filter through a sequence. Maybe I can move this map to constants file?

Reply:
Yeah it's more that we need this somewhere common most likely as it's probably used on many pages 👌

Reply:
@J-Son89 what do you mean by that? A sub to return that map?

Reply:
Whatever way you thinks make this more reuasbale. 
I think even a helper function id to network would be useful 👍 
```
#(get id-to-network %)
```
Mostly this functionality is broader in scope than this page.